### ISSUE TITLE
Dependency conflict with tty-prompt `necromancer`

### ISSUE THREAD

**Issue by danielpclark** [piotrmurach/tty-table#13]
```ruby
require 'tty-prompt'
# => true
require 'tty-table'

Gem::ConflictError: Unable to activate tty-table-0.7.0, because necromancer-0.4.0 conflicts with necromancer (~> 0.3.0)
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems/specification.rb:2288:in `raise_if_conflicts'
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems/specification.rb:1408:in `activate'
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems.rb:220:in `rescue in try_activate'
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems.rb:213:in `try_activate'
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:126:in `rescue in require'
	from /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from (irb):2
	from /usr/share/rvm/rubies/ruby-2.4.0/bin/irb:11:in `<main>'
```

**Comment by piotrmurach** [comment]
Thanks, I've updated `necromancer` recently to make it work correctly on Ruby 2.4. I will release new version shortly.

**Comment by piotrmurach** [comment]
Released `v0.8.0` that fixes this issue.
